search-service/src/test/kotlin/com/egm/stellio/search/service/AttributeInstanceServiceTests.kt

```diff
@@ -379,12 +382,12 @@ class AttributeInstanceServiceTests : WithTimescaleContainer, WithKafkaContainer
         }
 
         val temporalEntitiesQuery = gimmeTemporalEntitiesQuery(
-            TemporalQuery(
-                timerel = TemporalQuery.Timerel.BEFORE,
+            buildDefaultTestTemporalQuery(
+                timerel = Timerel.BEFORE,
                 timeAt = now,
                 aggrPeriodDuration = "PT1S",
                 aggrMethods = listOf(TemporalQuery.Aggregate.SUM),
-                lastN = 5
+                instanceLimit = 5,
```

Comment:
why remove the `lastN` here and change the test name ? Can't this be a new test case to add ?

Reply:
Now the size of the answer is define with instanceLimit not lastN (here we only test the size of the answer). If i do a test with lastN it is a completely different test. (which is way more complicated)